docs/src/main/sphinx/release/release-420.md

```diff
@@ -0,0 +1,66 @@
+# Release 420 (21 June 2023)
+
+## General
+
+* Add support for {func}`any_value` aggregation function. ({issue}`17777`)
+* Add support for underscores in numeric literals. ({issue}`17776`)
+* Add support for hexadecimal, binary, and octal numeric literals. ({issue}`17776`)
+* Deprecate the `dynamic-filtering.small-broadcast.*` and
+  `dynamic-filtering.large-broadcast.*` configuration properties in favor of
+  `dynamic-filtering.small.*` and `dynamic-filtering.large.*`. ({issue}`17831`)
+* Disallow using the root directory of an S3 bucket as a table location without
+  a trailing slash in the location name. ({issue}`17921`)
+* Fix query failure due to an exception from PartitionedOutputOperator. ({issue}`17861`)
+
+## Security
+
+* Introduce authorization rules to file-based access control for managing who
+  `ALTER ... SET AUTHORIZATION...` statements can set authorization for. ({issue}`16691`)
+* Remove the `legacy.allow-set-view-authorization` configuration property. ({issue}`16691`)
+
+## BigQuery connector
+
+* Fix direct web fetching of access tokens and correctly use the proxy when it
+  is enabled with the `bigquery.rpc-proxy.enabled` configuration property. ({issue}`17783`)
+
+## Delta Lake connector
+
+* Add support for [commenting](/sql/comment) on view columns. ({issue}`17773`)
+* Add support for recalculating all statistics with an `ANALYZE` statement. ({issue}`15968`)
+* Fix Parquet writer incompatibility with Apache Spark and Databricks Runtime. ({issue}`17978`)
+
+## Hive connector
+
+* Require admin role privileges to perform `ALTER ... SET AUTHORIZATION...`
+  statements when using standard security. ({issue}`16691`)
+* Improve query performance on partitioned Hive tables when table statistics are 
+  not present. ({issue}`17677`)
+* Fix Parquet writer incompatibility with Apache Spark and Databricks Runtime. ({issue}`17978`)
+* Fix reading from a Hive table when its location is the root directory of an S3
+  bucket. ({issue}`17848`)
+
+## Hudi connector
+
+* Fix Parquet writer incompatibility with Apache Spark and Databricks Runtime. ({issue}`17978`)
+* Fix failure when fetching table metadata for views. ({issue}`17901`)
+
+## Iceberg connector
+
+* Fix Parquet writer incompatibility with Apache Spark and Databricks Runtime. ({issue}`17978`)
+* Fix scheduling failure when dynamic filtering is enabled. ({issue}`17871`)
+
+## Kafka connector
+
+* Fix server startup failure when a Kafka catalog is present. ({issue}`17299`)
+
+## MongoDB connector
+
+* Add support for `RENAME COLUMN`. ({issue}`17874`)
```

Comment:
isnt that part of alter table and therefore has to be ALTER TABLE .. RENAME COLUMN